### ISSUE TITLE
Binary Classification of Grayscale Images with TF2

### ISSUE THREAD

**Issue by thrmotta** [google/automl#1043]
Have anyone managed to make a successful training on a Binary Classification of Grayscale images with TF2?

I did manage to start training, but after 20 epochs, Validation accuracy was close to 0.5 and loss was around 7 for both Train and Validation. Dataset is composed of 15k+  images and I can usually get a Validation Accuracy of 0.8+ with EfficientNetB0.

I'm not using the main_tf2 script though, as I wanted to reuse a previous code with ImageDataGenerator.

**Comment by halftru** [comment]
Did you ever fix this issue?
